client/src/pages/SearchPage.spec.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react'
+import SearchPage from './SearchPage'
+
+describe('SearchPage', () => {
+  it('renders a searchbar', () => {
+    render(<SearchPage />)
+    const searchbar = screen.getByText('Filter entries:')
+    expect(searchbar).toBeInTheDocument()
+  })
+})
```

Comment:
here you could test more ;) Like the onChange function and the rendering of the users.